package.json

```diff
@@ -30,6 +30,8 @@
     "@babel/preset-env": "^7.4.0",
     "assets-webpack-plugin": "^3.9.7",
     "babel-loader": "^8.0.0",
+    "browser-sync": "^2.18.13",
+    "browser-sync-webpack-plugin": "^2.2.2",
```

Comment:
`browser-sync` and `browser-sync-webpack-plugin` should only be declared as `devDependencies`.